### DISCUSSION TITLE
install composer in PhpStorm 2022

### DISCUSSION THREAD

**Discussion by MarcoGoeli** [composer/composer#11276] [Q&A / Support]
I installed php 8.2.1 and then composer-setup.exe to C:\php-8.2.1\php.exe.
Now I want to install the composer in PhpStorm 2022, but I can't do it. There are a few videos on Youtube, but it doesn't work with any instructions. Can someone help me.

Greetings Marco

**Comment by uuf6429** [comment]
First of all, PhpStorm is a different product, not related to Composer.

Secondly, what do you mean by installing composer in PhpStorm? What isn't working?
The typical process is is that you go to Preferences/Settings and select the composer binary (I guess that's what the tutorials and documentation would show).
